### ISSUE TITLE
attrs a requirement of fiona?

### ISSUE THREAD

**Issue by ecoon** [conda-forge/fiona-feedstock#97]
Issue: It appears to me that the fiona recipe needs to be updated to include attrs?  See below code snip to reproduce.

Note that after some searching, it looks like that dependency was introduced to Fiona about 9 days ago, but the source is updated here already!  Quick turn around or good automation.  Thanks!

    $> conda create -n fiona fiona
    $> conda activate fiona
    $> python -c "import fiona; print(fiona.__version__)"
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/Users/uec/codes/anaconda/3/5.3.0/conda_forge/envs/fiona/lib/python3.6/site-packages/fiona/__init__.py", line 83, in <module>
        from fiona.collection import BytesCollection, Collection
      File "/Users/uec/codes/anaconda/3/5.3.0/conda_forge/envs/fiona/lib/python3.6/site-packages/fiona/collection.py", line 9, in <module>
        from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
      File "fiona/ogrext.pyx", line 35, in init fiona.ogrext
      File "/Users/uec/codes/anaconda/3/5.3.0/conda_forge/envs/fiona/lib/python3.6/site-packages/fiona/path.py", line 6, in <module>
            import attr
    ModuleNotFoundError: No module named 'attr'

    $> conda install --name fiona attrs
    $> python -c "import fiona; print(fiona.__version__)"
    1.8a3

<br/>
Environment (note this is after installing attrs! (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /Users/uec/codes/anaconda/3/5.3.0/conda_forge/envs/fiona:
#
# Name                    Version                   Build  Channel
asn1crypto                0.24.0                py36_1003    conda-forge
attrs                     18.2.0                     py_0    conda-forge
blas                      1.0                         mkl
boost-cpp                 1.67.0               h3a22d5f_0    conda-forge
boto3                     1.9.20                     py_0    conda-forge
botocore                  1.12.20                    py_0    conda-forge
bzip2                     1.0.6                         1    conda-forge
ca-certificates           2018.8.24            ha4d7672_0    conda-forge
cairo                     1.14.12              he6fea26_5    conda-forge
certifi                   2018.8.24             py36_1001    conda-forge
cffi                      1.11.5           py36h5e8e0c9_1    conda-forge
click                     7.0                        py_0    conda-forge
click-plugins             1.0.4                      py_0    conda-forge
cligj                     0.5.0                      py_0    conda-forge
cryptography              2.3.1            py36hdffb7b8_0    conda-forge
cryptography-vectors      2.3.1                 py36_1000    conda-forge
curl                      7.61.1               h74213dd_2    conda-forge
docutils                  0.14                  py36_1001    conda-forge
expat                     2.2.5                hfc679d8_2    conda-forge
fiona                     1.8a3            py36hb00a9d7_0    conda-forge
fontconfig                2.13.1               hce039c3_0    conda-forge
freetype                  2.9.1                h6debe1e_4    conda-forge
freexl                    1.0.5                h470a237_2    conda-forge
gdal                      2.2.4            py36hb00a9d7_9    conda-forge
geos                      3.6.2                hfc679d8_3    conda-forge
geotiff                   1.4.2                h700e5ad_4    conda-forge
gettext                   0.19.8.1             h1f1d5ed_1    conda-forge
giflib                    5.1.4                h470a237_1    conda-forge
glib                      2.55.0               h464dc38_2    conda-forge
hdf4                      4.2.13               h951d187_2    conda-forge
hdf5                      1.10.2               hc401514_2    conda-forge
icu                       58.2                 hfc679d8_0    conda-forge
idna                      2.7                   py36_1002    conda-forge
intel-openmp              2019.0                      118
jmespath                  0.9.3                      py_1    conda-forge
jpeg                      9c                   h470a237_1    conda-forge
json-c                    0.12.1               h470a237_1    conda-forge
kealib                    1.4.9                h0bee7d0_2    conda-forge
krb5                      1.16.1               hbb41f41_0    conda-forge
libcurl                   7.61.1               hbdb9355_2    conda-forge
libdap4                   3.19.1               h18059cb_1    conda-forge
libedit                   3.1.20170329         haf1bffa_1    conda-forge
libffi                    3.2.1                hfc679d8_5    conda-forge
libgdal                   2.2.4                hdb23ba1_9    conda-forge
libgfortran               3.0.1                h93005f0_2
libiconv                  1.15                 h470a237_3    conda-forge
libkml                    1.3.0                hccc92b1_8    conda-forge
libnetcdf                 4.6.1                h039f2a5_8    conda-forge
libpng                    1.6.35               ha92aebf_2    conda-forge
libpq                     10.5                 hf16a0db_0    conda-forge
libspatialite             4.3.0a              h3b29d86_23    conda-forge
libssh2                   1.8.0                h5b517e9_2    conda-forge
libtiff                   4.0.9                he6b73bb_2    conda-forge
libxml2                   2.9.8                h422b904_5    conda-forge
mkl                       2019.0                      118
mkl_fft                   1.0.6                    py36_0    conda-forge
mkl_random                1.0.1                    py36_0    conda-forge
munch                     2.3.2                      py_0    conda-forge
ncurses                   6.1                  hfc679d8_1    conda-forge
numpy                     1.15.2           py36h6a91979_1
numpy-base                1.15.2           py36h8a80b8c_1
openjpeg                  2.3.0                h316dc23_3    conda-forge
openssl                   1.0.2p               h470a237_0    conda-forge
packaging                 18.0                       py_0    conda-forge
pcre                      8.41                 hfc679d8_3    conda-forge
pip                       18.1                  py36_1000    conda-forge
pixman                    0.34.0               h470a237_3    conda-forge
poppler                   0.67.0               h4d7e492_3    conda-forge
poppler-data              0.4.9                         0    conda-forge
postgresql                10.5                 ha408888_0    conda-forge
proj4                     4.9.3                h470a237_8    conda-forge
pycparser                 2.19                       py_0    conda-forge
pyopenssl                 18.0.0                py36_1000    conda-forge
pyparsing                 2.2.2                      py_0    conda-forge
pysocks                   1.6.8                 py36_1002    conda-forge
python                    3.6.6                h5001a0f_0    conda-forge
python-dateutil           2.7.3                      py_0    conda-forge
readline                  7.0                  haf1bffa_1    conda-forge
s3transfer                0.1.13                py36_1001    conda-forge
setuptools                40.4.3                   py36_0    conda-forge
shapely                   1.6.4            py36h164cb2d_1    conda-forge
six                       1.11.0                py36_1001    conda-forge
sqlite                    3.25.2               hb1c47c0_0    conda-forge
tk                        8.6.8                ha92aebf_0    conda-forge
urllib3                   1.23                  py36_1001    conda-forge
wheel                     0.32.1                   py36_0    conda-forge
xerces-c                  3.2.0                h5d6a6da_2    conda-forge
xz                        5.2.4                h470a237_1    conda-forge
zlib                      1.2.11               h470a237_3    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : fiona
    active env location : /Users/uec/codes/anaconda/3/5.3.0/conda_forge/envs/fiona
            shell level : 1
       user config file : /Users/uec/.condarc
 populated config files : /Users/uec/.condarc
          conda version : 4.5.11
    conda-build version : 3.15.1
         python version : 3.7.0.final.0
       base environment : /Users/uec/codes/anaconda/3/5.3.0/conda_forge  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/osx-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /Users/uec/codes/anaconda/3/5.3.0/conda_forge/pkgs
                          /Users/uec/.conda/pkgs
       envs directories : /Users/uec/codes/anaconda/3/5.3.0/conda_forge/envs
                          /Users/uec/.conda/envs
               platform : osx-64
             user-agent : conda/4.5.11 requests/2.19.1 CPython/3.7.0 Darwin/16.7.0 OSX/10.12.6
                UID:GID : 520249258:1551083765
             netrc file : None
           offline mode : False

```
</details>

**Comment by sgillies** [comment]
@ecoon sorry about this. I added attrs as a dependency in 1.8a3 and did not add it to the project's setup.py. 1.8b1 will have the fix, no later than Monday. The feedstock automation is indeed formidable: I saw it pick up on my Fiona tag and get to work before I'd even uploaded packages to PyPI.

**Comment by ecoon** [comment]
Ah, so I should have submitted an issue report to Fiona; the dependency will get added there then slurped in by the feedstock automatically?  Sorry for the issue in the wrong place then, I'm still learning the intricacies of how conda/channels/etc all work.

**Comment by ocefpaf** [comment]
@ecoon also, note that we wrongly published the alpha release in the `main` channel. I already fixed that, but you should not get `1.8a3` unless you explicitly install from the dev label. Sorry about that!
